Eth-Crowdfund/eth_crowdfund_be
# Mailer

## What's this PR do?
* Adds a mailer to send notifications when a request is made
* Pulls emails for all contributors to a campaign
## Where should the reviewer start? 
* src/mailers/mailer.py
## How should this be manually tested?
* Add a contributor with an email address you can check to a campaign OR
* Inside of src/views/RequestView change emails in line 32: `request_notification(emails, 'Test Link')`  to `[TempEmailAddress]`
* Using Postman send a post to create a new request for the campaign
* Check your email 
## Any background context you want to provide?
* Uses SendGrid, so this requires an API Key
* The email contains a dummy link, but will eventually point to the page where a contributor can vote on or view a request. 
## What are the relevant tickets?
*
